### PR TITLE
Feature/documentation

### LINKED ISSUE #21: Add proper documentation
Add Readme and describe usage, options, etc

### LINKED ISSUE #193: FragmentActivity.onResumeFragments()
[https://stackoverflow.com/questions/10114324/show-dialogfragment-from-onactivityresult](url)

### PR DESCRIPTION
**Pull request by WrobelKrakow**

### CODE REVIEW

File: core/src/main/java/pl/netigen/core/splash/CoreSplashFragment.kt [hunk 1]
```diff
@@ -15,7 +15,7 @@ import pl.netigen.extensions.observe
 import timber.log.Timber.d
 
 abstract class CoreSplashFragment : NetigenFragment(), GDPRDialogFragment.GDPRClickListener {
-    val splashVM: ISplashVM by activityViewModels<SplashVM> { coreMainActivity.viewModelFactory }
+    private val splashVM: ISplashVM by activityViewModels<SplashVM> { coreMainActivity.viewModelFactory }
```
Comment (WrobelKrakow) [inline core/src/main/java/pl/netigen/core/splash/CoreSplashFragment.kt:19]: Are we sure this should be private

**Review by WrobelKrakow — COMMENTED** [review]
On question

File: gms/src/main/java/pl/netigen/gms/payments/GMSPaymentsRepo.kt [hunk 2]
```diff
@@ -45,10 +48,10 @@ class GMSPaymentsRepo(
         return false
     }
 
-    override fun endConnection() {
+    // TODO: 10.05.2020 we should check if we should call this
```
Comment (WrobelKrakow) [inline gms/src/main/java/pl/netigen/gms/payments/GMSPaymentsRepo.kt:64]: Should we close connection onDestroy activity?

File: core-api/src/main/java/pl/netigen/coreapi/payments/IPayments.kt [hunk 3]
```diff
@@ -5,11 +5,62 @@ import android.content.Intent
...
+     * List of all payments sku available in Application
+     * for default it is only one default no-ads sku: [packageName] +".noads"
+     */
+    val inAppSkuList: List<String>
 
-    fun makePurchase(activity: Activity, netigenSkuDetails: NetigenSkuDetails)
-    fun endConnection()
+    /**
+     * [LiveData] with list of [NetigenSkuDetails], use this to get information of current available in app payments
+     * 01
```
Comment (rogalz) [inline core-api/src/main/java/pl/netigen/coreapi/payments/IPayments.kt]: What does that "01" mean ?

Comment (WrobelKrakow) [reply core-api/src/main/java/pl/netigen/coreapi/payments/IPayments.kt]: nothing:)

**Review by michal-bogucki — DISMISSED** [review]

**Review by michal-bogucki — APPROVED** [review]

**Pull request merged**